mesheryctl/internal/cli/root/system/check.go

```diff
@@ -513,8 +528,101 @@ func (hc *HealthChecker) runComponentsHealthChecks() error {
 }
 
 // runOperatorHealthChecks executes health-checks for Operators
-func (hc *HealthChecker) runOperatorHealthChecks() error {
+func (hc *HealthChecker) runOperatorHealthChecks(k8scontext string) error {
 	//TODO
+	url := hc.mctlCfg.GetBaseMesheryURL()
+	client := &http.Client{}
+	if hc.Options.PrintLogs {
+		log.Info("\nMeshery Operators \n--------------")
+	}
+
+	req, err := utils.NewRequest("GET", fmt.Sprintf("%s/api/system/kubernetes/contexts", url), nil)
+	if err != nil {
+		if hc.Options.PrintLogs {
+			log.Info("!! Authentication token not found. Please supply a valid user token. Login with `mesheryctl system login`")
+			// skip further as we failed to attach token
+			return nil
+		}
+		return errors.New("Authentication token not found. Please supply a valid user token. Login with `mesheryctl system login`")
+	}
+	var pages *models.MesheryK8sContextPage
+	var contexts []*models.K8sContext
+	resp, err := client.Do(req)
+	if err != nil || resp.StatusCode != 200 {
```

Comment:
Are we continuing the execution even in case of an error? there might not be a response body to process